GoogleBooks/src/functions/helpers.js

```diff
@@ -0,0 +1,17 @@
+export const fetchBooks = async (query) => {
+  if (query) {
+    try {
+      let acceptableHeader = "application/json";
+      let fetchURL = `https://www.googleapis.com/books/v1/volumes?q=${query}&key=AIzaSyCZ15z_YoUnk6OVD-OVBNIMgAIC2XQzz-o`;
+      const response = await fetch(fetchURL, {
+        headers: { Accept: acceptableHeader },
+      });
+
+      const data = await response.json();
+
+      return data.items;
+    } catch (error) {
+      console.log(error);
```

Comment:
If you throw this error here you can use it in the catch in CardHolder to set some kind of Error state

Reply:
```js
throw new Error('Unable to search for books')
```